actions/build-log-to-junit/action.yml

```diff
@@ -1,14 +1,14 @@
-name: 'Generate JUnit Report'
-description: 'Generate JUnit report from make build output'
+name: 'Verify build log'
+description: 'Verify build log, if all targets are built'
```

Comment:
"Verify" usually means that we get a verdict either "verified" or "verification failed". Here we generate a summary in JUnit format, and was it success or failure is establish in later steps. 
Thus, I think, prev name was better. If you want to explicitly mention we process build logs here, make it "Generate JUnit Report from build log"

Reply:
My point is that, when I first saw the CI job name, I thought it is parsing test logs.

Reply:
Okey, reverted it now